roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -66,25 +66,37 @@ def _print_failure_summary(self):
 # Status: permanently disabled unless Ansible's API changes.
 # pylint: disable=protected-access
 def _format_failure(failure):
-    '''Return a list of pretty-formatted lines describing a failure, including
-    relevant information about it. Line separators are not included.'''
+    '''Return a list of pretty-formatted text entries describing a failure, including
+    relevant information about it. Line breaks can be included but not trailing newlines.'''
```

Comment:
- :+1: s/lines/text entries is good, we now say that each entry is arbitrary text (in fact, it already was and already contained multiple lines, e.g., the pretty-formatted JSON).
- :-1: the second change doesn't add much clarity for me. There can certainly be trailing newlines, e.g., if we want to add extra space between sections. "Separator" is the term we find in the Python docs for splitting and joining text -- https://docs.python.org/3/library/stdtypes.html#str.split